app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        total_wash_price = 0
+        for car in cars:
+            if self.wash_single_car(car):
+                total_wash_price += self.calculate_washing_price(car)
+                car.clean_mark = self.clean_power
+
+        return total_wash_price
+
+    def calculate_washing_price(self, car: Car) -> float:
+        multiplication_top = (car.comfort_class * self.average_rating
+                              * (self.clean_power - car.clean_mark))
+        return round(multiplication_top / self.distance_from_city_center, 1)
```

Comment:
Code Style issue: According to the checklist, if you have some long math, you can split it onto additional variables. This will make your code more readable and maintainable. Consider splitting the calculation in 'calculate_washing_price' method into several steps. Here's how you might do it:

```python
comfort_rating_product = car.comfort_class * self.average_rating
power_difference = self.clean_power - car.clean_mark
multiplication_top = comfort_rating_product * power_difference
return round(multiplication_top / self.distance_from_city_center, 1)
```